website/src/components/TutorialComponents/CodeBlock.tsx

```diff
@@ -76,37 +96,49 @@ const CodeEditor = (props) => {
                     getTokenProps,
                     style: _style,
                 }) => (
-                    <pre
-                        className={_className}
-                        style={{
-                            margin: 0,
-                            outline: "none",
-                            padding: 10,
-                            fontFamily: "inherit",
-                            ...(!props.className || !props.theme ? {} : _style),
-                        }}
-                        ref={editorRef}
-                        spellCheck="false"
-                    >
-                        {tokens.map((line, lineIndex) => (
-                            // eslint-disable-next-line react/jsx-key
-                            <div {...getLineProps({ line, key: `line-${lineIndex}` })}>
-                                {line
-                                    .filter((token) => !token.empty)
-                                    .map((token, tokenIndex) => (
-                                        // eslint-disable-next-line react/jsx-key
-                                        <span
-                                            {...getTokenProps({ token, key: `token-${tokenIndex}` })}
-                                        />
-                                    ))}
-                                {"\n"}
+                    <div style={{ display: "flex" }}>
+                        {props.showLineNumbers ?
```

Comment:
nit: destructure props arugments at the start of the component
nit: you can also do 

    {props.showLineNumbers && <div ....></div>}

Reply:
Resolved in #98